### PR TITLE
Update monitoring guide for OCP 3.1x, add additional escape chars

### PR DESCRIPTION
**Pull request by themoosman**
#### What is this PR About?
Update monitoring guide for OCP 3.1x. Probably need to discuss if these commands should be considered "best practice." Putting something out there to start a conversation.

#### How should we test or review this PR?
Run the new commands on any 3.1x cluster.

#### Is there a relevant Trello card or Github issue open for this?
Issue #108

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----

- [ x] Have you followed the [contributing guidelines](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md)?
- [ x] Have you explained what your changes do, and why they add value to the uncontained.io guides?

-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: site/content/articles/openshift-monitoring-guide.adoc [hunk 1]
```diff
@@ -48,35 +48,37 @@ Docker is an essential component of an OpenShift environment. The overall health
 
 |Docker Daemon |Check that docker is running on a system | `systemctl is-active docker`
 
-|Docker Storage|Check that docker's storage has adequate space | `echo $(echo \"$(docker info 2>/dev/null \| awk '/Data Space Available/ {print $4}') / $(docker info 2>/dev/null \| awk '/Data Space Total/ {print $4}')\" \| bc -l) '>' 0.3 \| bc -l`
+|Docker Storage|Check that docker's storage has adequate space | `echo $(echo \"$(docker info 2>/dev/null  &#124; awk '/Data Space Available/ {print $4}') / $(docker info 2>/dev/null  &#124; awk '/Data Space Total/ {print $4}')\"  &#124; bc -l) '>' 0.3  &#124; bc -l`
```
Comment (sabre1041) [inline site/content/articles/openshift-monitoring-guide.adoc]: What values are these supposed to return. In my test queries, both returned `1` with an exit code of 0

File: site/content/articles/openshift-monitoring-guide.adoc [hunk 2]
```diff
@@ -48,35 +48,37 @@ Docker is an essential component of an OpenShift environment. The overall health
...
 === Nodes & Masters
 
 .Node,Master checks
-[width="100%",cols="3,6,2,10",options="header"]
-|=========================================================
-|Check Name |Description | Relevant Hosts| Sample Alerting Logic
+[width="100%",cols="15%,30%,10%,5%,40%",options="header"]
+|===
+| Check Name | Description | Relevant Hosts | OCP Version | Sample Alerting Logic
+.2+|Etcd Service .2+|Check that etcd is active .2+|Masters | \<= 3.9 | `systemctl is-active etcd`| >= 3.10 | `oc get pods -n kube-system --no-headers -o=custom-columns=POD:.metadata.name,STATUS:.status.phase &#124; grep -i "master-etcd" &#124; grep -i "running" &#124;  if [ $( wc -l) -eq 3 ]; then exit 0; else exit 1; fi`
```
[the remark this answers is not in the document]
Comment (themoosman) [reply site/content/articles/openshift-monitoring-guide.adoc]: Added code to count the number of etcd pods.

File: site/content/articles/openshift-monitoring-guide.adoc [hunk 3]
```diff
@@ -48,35 +48,37 @@ Docker is an essential component of an OpenShift environment. The overall health
...
 
-|Master API Service (single master) |Check that the Master API Service is active|Masters | `systemctl is-active atomic-openshift-master`
+.2+|Master API Service (multi-master) .2+|Check that the Master API Service or pods are active .2+|Masters |  \<= 3.9 |`systemctl is-active atomic-openshift-master-api`| >= 3.10 | `oc get pods -n kube-system --no-headers -o=custom-columns=POD:.metadata.name,STATUS:.status.phase &#124; grep -i "master-api" &#124; grep -i "running" &#124;  if [ $( wc -l) -eq 3 ]; then exit 0; else exit 1; fi`
 
-|Master API Service (multi-master) |Check that the Master API Service is active|Masters | `systemctl is-active atomic-openshift-master-api`
+.2+|Master Controllers Service (multi-master) .2+| Check that the Master Controllers Service or pods are active .2+|Masters | \<= 3.9 | `systemctl is-active atomic-openshift-master-controllers`| >= 3.10 | `oc get pods -n kube-system --no-headers -o=custom-columns=POD:.metadata.name,STATUS:.status.phase &#124; grep -i "master-controller" &#124; grep -i "running" &#124; if [ $( wc -l) -eq 3 ]; then exit 0; else exit 1; fi`
```
[the remark this answers is not in the document]
Comment (themoosman) [reply site/content/articles/openshift-monitoring-guide.adoc]: Added code to count the number of etcd pods.

File: site/content/articles/openshift-monitoring-guide.adoc [hunk 4]
```diff
@@ -48,35 +48,37 @@ Docker is an essential component of an OpenShift environment. The overall health
...
 
-|Master Controllers Service (multi-master) | Check that the Master Controllers Service is active|Masters | `systemctl is-active atomic-openshift-master-controllers`
+.2+|Node Service .2+|Check that the node service is active .2+| All Nodes |  \<= 3.9 |`systemctl is-active atomic-openshift-node`| >= 3.10 | `systemctl is-active atomic-openshift-node`
 
-|Node Service |Check that the node service is active | All Nodes| `systemctl is-active atomic-openshift-node`
+.2+|Node Storage .2+|Check that the node's local data storage volume is not too full .2+| All Nodes | \<= 3.9 | `echo "$(lvs &#124; awk '/origin/ {print $5}') > 70" &#124; bc`| >= 3.10 | `echo "$(lvs &#124; awk '/origin/ {print $5}') > 70" &#124; bc`
```
[the remark this answers is not in the document]
Comment (themoosman) [reply site/content/articles/openshift-monitoring-guide.adoc]: Fixed awk parsing and added alternative method using `df`

File: site/content/articles/openshift-monitoring-guide.adoc [hunk 5]
```diff
@@ -87,11 +89,11 @@ Many OpenShift components expose HTTP based endpoints for interrogating the heal
 |=========================================================
 |Check Name |Description |Sample Alerting Logic
 
-|OpenShift Master API Server |Check the health of a master API Endpoint| `curl -H "Authorization: Bearer $(oc whoami -t)" https://console.c1-ocp.myorg.com:8443/healthz \| grep ok`
+|OpenShift Master API Server |Check the health of a master API Endpoint| `curl -H "Authorization: Bearer $(oc whoami -t)" https://console.c1-ocp.myorg.com:8443/healthz  &#124; grep ok`
```
[the remark this answers is not in the document]
Comment (themoosman) [reply site/content/articles/openshift-monitoring-guide.adoc]: Remove auth.

File: site/content/articles/openshift-monitoring-guide.adoc (hunk 1 above)
Comment (themoosman) [reply site/content/articles/openshift-monitoring-guide.adoc]: Had to split this into 2 different commands.  1 command is for devicemapper the other is for overlay2.  The original command would not work on a system configured with overlay2.

**Comment by themoosman** [comment]
Updated guide.

**Pull request merged**